packages/studio-plugin/tests/__fixtures__/PageFile/brokenCssImport.tsx

```diff
@@ -0,0 +1,22 @@
+import "@yext/search-ui-react/bundle.css";
+import ComplexBanner from "../ComponentFile/ComplexBanner";
+import { GetPath, TemplateConfig, TemplateProps } from "@yext/pages";
+
+export const config: TemplateConfig = {
+  stream: {
+    $id: "studio-stream-id",
+    localization: { locales: ["en"] },
+    filter: { entityTypes: ["location"] },
+    fields: ["title", "slug"],
+  },
+};
+
+export const getPath: GetPath<TemplateProps> = ({
+  document,
+}: TemplateProps) => {
+  return document.slug;
+};
+
+export default function BrokenCss({ document }: TemplateProps) {
```

Comment:
nit: can this be called unsupported CSS instead of broken? it's not that there's something wrong with the CSS. we just don't support it right now